examples/src/main/java/com/google/instrumentation/trace/MultiSpansScopedTracing.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2017, Google Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.instrumentation.trace;
+
+import com.google.instrumentation.common.NonThrowingCloseable;
+
+/**
+ * Example showing how to create a child {@link Span} using {@link ScopedSpan}, install it in the
+ * current context, and add annotations.
+ */
+public final class MultiSpansScopedTracing {
```

Comment:
maybe an example of async. For example, the scoping part is implicitly attaching the span as current. sometimes we need to set current from a span created on another thread.